manim/utils/space_ops.py

```diff
@@ -766,3 +767,11 @@ def earclip_triangulation(verts: np.ndarray, ring_ends: list) -> list:
 
     meta_indices = earcut(verts[indices, :2], [len(indices)])
     return [indices[mi] for mi in meta_indices]
+
+
+def perpendicular_bisector(line: Sequence[Sequence[float]], norm_vector=OUT):
```

Comment:
Can you specify the return type, and add a doc string describing the input and outputs (explaining that the two lists are the endpoints of the line)

Reply:
Perhaps another feature good to have is to be able to specify whether you want the output in endpoint style or position+direction vectors style, but that can come in a future PR.

Reply:
> specify the return type, and add a doc string describing the input and outputs

Sure

> specify whether you want the output in endpoint style or position+direction vectors style

I also think that's a good idea for another PR